.travis.yml

```diff
@@ -1,52 +1,48 @@
 language: python
 cache: pip
 
-matrix:
+python:
+  - 3.5
+  - 3.6
+  - 3.7
+  - 3.8
+env:
+  - DJANGO=1.11
+  - DJANGO=2.2
+  - DJANGO=3.0
+  - DJANGO=master
+
+jobs:
   fast_finish: true
+  allow_failures:
+    - env: DJANGO=master
   include:
-    - env: TOXENV=black
+    - python: 3.8
+      env: TOXENV=black
```

Comment:
I think you'll have to specify the Python version for each of the extra environments, not only for black, to make sure we run them on 3.8, e.g. flake8 and isort are run on 3.5 at the moment.